api/v1alpha1/types.go

```diff
@@ -16,11 +16,44 @@ limitations under the License.
 
 package v1alpha1
 
+import (
+	platformv1alpha1 "github.com/openshift/api/platform/v1alpha1"
+)
+
 var (
 	TypeApplied = "Applied"
 
 	ReasonSourceFailed    = "SourceFailed"
+	ReasonUnpackPending   = "ApplyPending"
 	ReasonApplyFailed     = "ApplyFailed"
 	ReasonApplySuccessful = "ApplySuccessful"
 	ReasonApplyPending    = "ApplyPending"
 )
+
+const (
+	SourcedBundleAnnotation = "platform.openshift.io/sourced-bundle"
+)
+
```

Comment:
Is this related to the convo we had about how to gate sourcing if we've already sourced successfully? If so, add a TODO to switch over to gating based on BD presence? Or did you discover an issue with that approach?

Reply:
I see we _are_ already gating on BD presence. Is this dead code?

Reply:
Yup, this was dead code. Removed in https://github.com/openshift/platform-operators/pull/41/commits/28b4316bc8c76cdb4d61f41877fbe6351ac50f39.